docs/using_tags.md

```diff
@@ -127,7 +138,7 @@ Alternatively if you have a location(s) that you render inline & want invalidate
 Fo PHP usage, FOSHttpCache exposes `fos_http_cache.handler.tag_handler` service which lets you add tags to a response:
 ```php
     /** @var \FOS\HttpCache\Handler\TagHandler $tagHandler */
-    $tagHandler->addTags(['relation-33', 'relation-44']);
+    $tagHandler->addTags(['r33', 'r44']);
```

Comment:
A suggestion would be to introduce some more abstraction here so that you specify what kind of tags to add and that http-bundle based on that creates the correct key..  For instance:

`
     $tagHandler->addRelationTags([33, 44]);
`

This way we can change tag names without affecting 3rd party code ( if 3rd party code uses this abstraction instead of hard-coding the tag names